### DISCUSSION TITLE
Type Hints in Examples

### DISCUSSION THREAD

**Discussion by jgarte** [Abjad/abjad#1567] [Ideas]
Hi,

What do you think if we were to type annotate the _Examples_ section?

For example, in the [magic square](https://abjad.github.io/examples/magic-square-from-twelve-tone-row.html#magic-square-from-twelve-tone-row) example, we could write the following code that returns the transpositions of a tone row as:

```python
import abjad as aj

def make_transpositions(row: aj.TwelveToneRow) -> list[aj.TwelveToneRow]:
    transpositions = []
    for pitch_class in row.invert():
        n = pitch_class.number - row[0].number
        transposition = row.transpose(n)
        transpositions.append(transposition)
    return transpositions
```

I think this would document more clearly for newcomers on what they are supposed to be passing in to functions in the _Examples_ documentation.

Just a thought. 

I'd be happy to submit pull requests adding this if you think it is a good idea?

**Comment by trevorbaca** [comment]
Abjad itself is typehinted, because it's complex, and a library. But there's really no reason for Abjad users to typehint their own scores. So, no reason to typehint the examples, either.
